opentech/apply/funds/templates/funds/includes/review_table.html

```diff
@@ -1,33 +1,56 @@
 {% load review_tags %}
 <table class="reviews-sidebar js-reviews-sidebar">
-    {% if staff_reviews or object.staff_not_reviewed %}
-        {% if staff_reviews %}
-            <tr class="tr tr--subchild light-grey-bg">
-                <th colspan="2"></th>
-                <th>{{ object.reviews.submitted.recommendation|traffic_light }}</th>
-                <th>{{ object.reviews.submitted.get_score_display|default_if_none:'-' }}</th>
-            </tr>
-            {% include 'funds/includes/review_table_row.html' with reviews=staff_reviews %}
-        {% endif %}
+    {% if top_reviews %}
+        <tr class="tr tr--subchild light-grey-bg">
+            <th colspan="2"></th>
+            <th>{{ recommendation|traffic_light }}</th>
+            <th></th>
+        </tr>
+    {% endif %}
 
-        {% if object.staff_not_reviewed %}
-            {% include 'funds/includes/review_table_row.html' with reviews=object.staff_not_reviewed missing=True %}
-        {% endif %}
-    {% else %}
+    {% if reviews_block.role_reviewed or reviews_block.role_not_reviewed %}
```

Comment:
We dont need the if statements here if the for loop is empty we dont render anything

Reply:
Yes, indeed